docs/eventing/migration/ping.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Migrating to the PingSource"
+weight: 20
+type: "docs"
+aliases:
+   - /docs/eventing/ping.md
+---
+
+The _deprecated_ `CronJobsource` should be converted to the newer `PingSource`.
+
+Imagine you have a `CronJobSource` running in your cluster that emits its events to a Knative Serving Service.
```

Comment:
Don't say "Imagine...", it's too fluffy.
Instead, something like:
"The YAML file for a `CronJobSource` that emits events to a Knative service will look similar to this:"